app/ProjectAanvraag/ProjectAanvraagUrl.php

```diff
@@ -14,10 +14,10 @@ public static function getForIntegration(Integration $integration): string
     {
         $idToken = Session::get('id_token');
 
-        return self::getStatusBaseUri($integration->status) . 'project/' . $integration->id . '/widget/?idToken=' . $idToken;
+        return self::getBaseUrlForIntegrationStatus($integration->status) . 'project/' . $integration->id . '/widget/?idToken=' . $idToken;
     }
 
-    public static function getStatusBaseUri(IntegrationStatus $integrationStatus): mixed
+    public static function getBaseUrlForIntegrationStatus(IntegrationStatus $integrationStatus): mixed
```

Comment:
Now I also notice the return type, this should be `string`

Reply:
[Change return type](https://github.com/cultuurnet/publiq-platform/pull/906/commits/337bf6b4c4e9616b49cda3f7799cb85143df7295)